### DISCUSSION TITLE
car takes 7 hours to fall asleep

### DISCUSSION THREAD

**Discussion by Holger237237** [bassmaster187/TeslaLogger#1131] [Q&A]
After driving or charging, Teslalogger does not access the vehicle for 21 minutes to ensure that it can fall asleep. Every minute the message appears in the log file from 0-20, followed by the message "asleep".
That's how it should be. In my case, however, the minute counter is not incremented every minute but only every 20 minutes.

[[19.11.2023 11:58:16 : #4[Car_4:24]: STOP communication with Tesla Server to enter sleep Mode! https://teslalogger.de/faq-1.php](https://teslalogger.de/faq-1.php)
19.11.2023 12:00:11 : #4[Car_4:24]: Waiting for car to go to sleep 0
19.11.2023 12:20:23 : #4[Car_4:24]: Waiting for car to go to sleep 1
19.11.2023 12:40:34 : #4[Car_4:24]: Waiting for car to go to sleep 2
19.11.2023 13:00:45 : #4[Car_4:24]: Waiting for car to go to sleep 3
19.11.2023 13:20:56 : #4[Car_4:24]: Waiting for car to go to sleep 4
19.11.2023 13:41:14 : #4[Car_4:24]: Waiting for car to go to sleep 5
19.11.2023 14:01:25 : #4[Car_4:24]: Waiting for car to go to sleep 6
19.11.2023 14:21:37 : #4[Car_4:24]: Waiting for car to go to sleep 7
19.11.2023 14:41:48 : #4[Car_4:24]: Waiting for car to go to sleep 8](url)

Means that "asleep" is only set after 7 hours. The status during this time is always "suspended". Communication is not available until then. In fact, the vehicle has already fallen asleep normally.
What could be the reason for this? I suspect a connection with the fact that I log 4 vehicles.

Version: 1.55.0.0

Thanks for helping
Holger

**Comment by bassmaster187** [comment]
Car model?
What AP do you have? 1 / 2 / 2.5 or 3? 
Did you do a MCU upgrade?
Can you please post Grafana dashboard "Status"?

**Comment by Holger237237** [comment]
-Car Model: 3x Model 3, 1x Model Y
-AP?
-MCU?
- Here the Status of one of them: 
![image](https://github.com/bassmaster187/TeslaLogger/assets/94004539/0e1e293b-a2bb-4880-acf4-09a1907fc663)

**Comment by bassmaster187** [comment]
My Model 3 sleeps very well. I saw a couple of days the car is downloading new maps. Maybe this takes forever if you have bad wifi coverage. You can also try to reset your car. Maybe it is in any endless loop?

![image](https://github.com/bassmaster187/TeslaLogger/assets/6816385/1bdc3c0b-6754-4f0a-9831-857ccf8e8c15)

**Comment by Holger237237** [comment]
All my cars also sleep very well. But Teslalogger does not know this because it is still in waiting position (not only for this cars. It is just an example):

[19.11.2023 11:58:16 : #4[Car_4:24]: STOP communication with Tesla Server to enter sleep Mode! https://teslalogger.de/faq-1.php](https://teslalogger.de/faq-1.php)
19.11.2023 **12:00:11** : https://github.com/bassmaster187/TeslaLogger/issues/4[Car_4:24]: Waiting for car to go to sleep 0
19.11.2023 **12:20:23** : https://github.com/bassmaster187/TeslaLogger/issues/4[Car_4:24]: Waiting for car to go to sleep 1
19.11.2023 **12:40:34** : https://github.com/bassmaster187/TeslaLogger/issues/4[Car_4:24]: Waiting for car to go to sleep 2
19.11.2023 **13:00:45** : https://github.com/bassmaster187/TeslaLogger/issues/4[Car_4:24]: Waiting for car to go to sleep 3
19.11.2023 **13:20:56** : https://github.com/bassmaster187/TeslaLogger/issues/4[Car_4:24]: Waiting for car to go to sleep 4

The cars are all fine. Teslalogger isn't.

e.g. I miss the log-line at 12:01:
19.11.2023 12:**01**:11 : https://github.com/bassmaster187/TeslaLogger/issues/4%5BCar_4:24%5D: Waiting for car to go to sleep 1

it takes 19 minutes more
